lib/behaviors/bootstrapvalidator-behavior.js

```diff
@@ -1,30 +1,43 @@
 'use strict';
 
-var Marionette = require('backbone.marionette'),
-  _ = require('underscore');
+(function(root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define([
+            'marionette',
+            'underscore',
+            'bootstrapValidator'
+        ], factory);
+    } else if (typeof exports === 'object') {
+        module.exports = factory(
+            require('backbone.marionette'),
+            require('underscore'),
+            require('bootstrapValidator')
+        );
+    }
+})(this, function(Marionette, _, BootstrapValidator) {
 
-require('bootstrapValidator');
+    return Marionette.Behavior.extend({
+        onRender: function() {
```

Comment:
does it still work on your end? should we update to [OpenValidator](https://github.com/OpenValidator/openvalidator)?